### ISSUE TITLE
Weird training accuracy curve 

### ISSUE THREAD

**Issue by gongliyu** [PetarV-/GAT#7]
Hi,

I observed that the training accuracy is very low all the time during the training process. The values is around 50%. Although validation accuracy reaches around 83%, this is still weird. I tried Kipf's GCN code, it can reach above 90% training accuracy, which is much normal.

Does anybody notice this?

**Comment by PetarV-** [comment]
Hi Liyu,

This is caused by the strong dropout on the alpha_ij values. It causes the network to mispredict more often during training, as the graph neighbourhoods get quite variable; however this makes it less prone to overfitting on small training datasets such as Cora.

Hope that helps!

Thanks,
Petar

**Comment by gongliyu** [comment]
Hi Petar,

Got it. Thank you very much for the explanation.

Best,
Liyu
